code/src/app/unitenew/unite.module.ts

```diff
@@ -9,6 +9,7 @@ import { AdminBranch } from './branch/admin/admin.branch';
 
 import { GlobalConfig } from './configs/global.configs';
 import { Menues } from './configs/menus.configs';
```

Comment:
@ttplsudhir - Use correct spelling. It should be `Menus`

Reply:
@sushan05 Yeah will update code accordingly